README.md

```diff
@@ -1,14 +1,14 @@
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
 
+# ContactCS
+
 ![Ui](docs/images/Ui.png)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
-  Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-  * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org/#contributing-to-se-edu) for more info.
+**ContactCS** is designed for NUS Computer Science freshmen to efficiently manage and locate important contact details, including those of professors, teaching assistants, classmates, school offices, and more.
+
+For the detailed documentation of this project, see the **[ContactCS Product Website](https://ay2425s1-cs2103t-f12-1.github.io/tp)**.
+
+### About us
+
+**ContactCS** is developed by a team of 5 students from NUS Computer Science as a project for a course on Software Engineering (CS2103T). You can find out more about us [here](https://ay2425s1-cs2103t-f12-1.github.io/tp/AboutUs.html).
+
```

Comment:
The description here is well written as it mentions the specific target user for our application and the help it provides to freshmen while remaining short and concise.